vignettes/getting_started.Rmd

```diff
@@ -40,55 +37,107 @@ Most of this can be done with a single function call to `automateDataPreparation
+The **values sheet** contains all possible values for each subitem and scoring information for each value (e.g., whether it is true, false, or missing). There are several types of missing values.
+
+```{r values sheet}
+kable(inputMinimal$values[which(inputMinimal$values$subunit %in% subitems), ])
+```
+
+<!-- talk more about values in the data for different item types (mc, cr) and about missing values in the data. missing values do not need to be the same for each variable in the data.-->
+
+The **subunits sheet** contains information about the relationship between subitems and items. This information is used to aggregate subitems to items. If recoded subitems should be named differently than the unrecoded variables in the original data, this can be specified via `subunitRecoded`. The only type of aggregation currently supported by `eatPrep` is to count the number of correct subitems per item.
 
---> talk more about values in the data for different item types (mc, cr) and about missing values in the data. missing values do not need to be the same for each variable in the data.
+```{r subunits sheet}
+kable(inputMinimal$subunits[which(inputMinimal$subunits$subunit %in% subitems), ])
+```
 
+**unitrecodings** contains the same information as the values sheet, but for aggregated items. Here, the value on an item is the aggregated number of correct subitems and the recode value gives a threshold of how many subitems need to be solved to obtain a credit for the item. In the example, I12 has 3 subitems, and one receives a credit if all subitems are solved correctly.
 
-The subunits sheet contains information about the relationship between subitems and items. This information is used to aggregate subitems to items. If recoded subitems should be named differently than the unrecoded variables in the original data, this can be specified via `subunitRecoded`. The only type of aggregation currently supported by `eatPrep` is to count the number of correct subitems per item.
+```{r unitRecodings}
+kable(inputMinimal$unitRecodings)
+```
 
-unitrecodings contains the same information as the values sheet, but for aggregated items. Here, the value on an item is the aggregated number of correct subitems and the recode value gives a threshold of how many subitems need to be solved to obtain a credit for the item. In the example, I12 has 3 subitems, and one receives a credit if all subitems are solved correctly.
+The **minimal use case** is that each subitem corresponds to an item. In this case, the value sheet contains all information needed for recoding the items (apart from possibly renaming the recoded items).
 
-The minimal use case is that each subitem corresponds to an item. In this case, the value sheet contains all information needed for recoding the items (apart from possibly renaming the recoded items).
 
 ## input data
 
```

Comment:
what about showing the functions readDeamonXlsx() and readMerkmalXlsx()? their example .xlsx files are in inst/extdata. but there already is `inputDat`.
What kind of large data set should I construct? is inputDat the one? how do I construct that from scratch/where do I get that from?

Reply:
Showing the functions readDeamonXlsx() and readMerkmalXlsx() using their example .xlsx files are in inst/extdata is a very good idea!! inputDat is the corresponding data, I think, no further construction of artificial data should be necessary for illustrational purposes...